src/commands/voice.js

```diff
@@ -0,0 +1,79 @@
+/*
+    Copyright (C) 2023 Alexander Emanuelsson (alexemanuelol)
+    Copyright (C) 2023 Squidysquid1
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+    https://github.com/alexemanuelol/rustplusplus
+
+*/
+
+const Builder = require('@discordjs/builders');
+const {joinVoiceChannel, getVoiceConnection} = require('@discordjs/voice');
+const DiscordMessages = require('../discordTools/discordMessages.js');
```

Comment:
Leave a space between package imports and local files. Ocd thing.

`const Builder = require('@discordjs/builders');
const {joinVoiceChannel, getVoiceConnection} = require('@discordjs/voice');

const DiscordMessages = require('../discordTools/discordMessages.js');`